tools/replay_allocation_map.cpp

```diff
@@ -66,7 +66,7 @@ std::istream& operator>>(std::istream& str, CSVRow& data)
 {
```

Comment:
This should be removed.  It is not being maintained and is not needed (UM-358)

Reply:
I removed the TODO below and we discussed putting this in a separate PR. Resolving for now.